### PR TITLE
Add an undeploy cursor and make construction yards use it

### PR DESCRIPTION
**Pull request by abcdefg30**
See #29.

### CODE REVIEW

File: cursors.yaml [hunk 1]
```diff
@@ -101,7 +101,7 @@ Cursors:
 			Length: 9
 		deploy-blocked:
 			Start: 119
-		undeploy:
+		undeploy-unused:
```
Comment (Phrohdoh) [inline cursors.yaml]: Why keep unused markup?

Comment (abcdefg30) [inline cursors.yaml]: Removed it.

**Comment by abcdefg30** [comment]
Updated again.

File: cursors.yaml (hunk 1 above)
Comment (abcdefg30) [inline cursors.yaml]: Readded it as requested, just as `undeploy-silver` now, as it's color is silver. I tend to agree to @Mailaender that this a) won't hurt and b) the artwork is still there and could be used.

**Comment by Mailaender** [comment]
Nicely done. Thanks to @x-a-n-a-x.

**Pull request merged**